Remastered NPC Holograms (#2243)
BetonQuest/BetonQuest: docs/Documentation/Compatibility.md

### NPC Holograms

If Citizens is also installed then you can have holograms configured relative to an NPC.

> If you have moving NPCs (waling around) then you can have the holograms follow them by setting `follow: true`,

Reviewer: ```suggestion
If you have moving NPCs (walking around) then you can have the holograms follow them by setting `follow: true`,
```

but this will cause a lot of updates to the holograms and may cause lag if used on a lot of NPCs.

> So only set this to true on holograms that are on NPCs that are actually moving.

Reviewer: ```suggestion
So only set this to true for holograms with an NPC that actually moves.
```
